recipe/conda_build_config.yaml

```
@@ -2,6 +2,11 @@ rust_arch:
   - x86_64-unknown-linux-gnu       # [linux64]
   - s390x-unknown-linux-gnu        # [s390x]
   - aarch64-unknown-linux-gnu      # [aarch64]
-  - powerpc64le-unknown-linux-gnu  # [ppc64le]
   - x86_64-apple-darwin            # [osx and x86_64]
   - aarch64-apple-darwin           # [osx and arm64]
+macos_min_version:
+  - 10.12 # [osx and x86_64]
+MACOSX_DEPLOYMENT_TARGET:
+  - 10.12 # [osx and x86_64]
+CONDA_BUILD_SYSROOT:
+  - /opt/MacOSX10.12.sdk        # [osx and x86_64]
```

```suggestion
macos_min_version:  # [osx and x86_64]
  - 10.12  # [osx and x86_64]
MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
  - 10.12  # [osx and x86_64]
CONDA_BUILD_SYSROOT:  # [osx and x86_64]
  - /opt/MacOSX10.12.sdk  # [osx and x86_64]

```
Because you change for `osx and x86_64` only

---

That is fine, the default minimum version on arm64 is 11 something.